degirum_tools/figure_annotator.py

```diff
@@ -210,9 +470,24 @@ def __init__(
             self.num_vertices == FigureAnnotatorType.QUADRILATERAL.value
         )
         self.results_file_name = results_file_name
+        self.save_path = ""
+
+        # Constants
+        self.line_width = 1
 
         self.root = tk.Tk()
         self.root.title(f"{self.figure_type.capitalize()} Annotator")
+        self.root.geometry("700x500")
+
+        # Override the close button event
+        self.root.protocol("WM_DELETE_WINDOW", self.on_close)
+
+        # Set window icon
+        self.icon_image = ImageTk.PhotoImage(file="assets/logo.png")
```

Comment:
How this file will appear in wheel installation?
If you really need it, you do this:
1. Include it into the wheel installation. For example, into assets subdir of root package install dir.
2. Construct full path to this file using the path of the current source file as a bas